src/arch/arm/armlib/exception_handler.c

```diff
@@ -10,14 +10,14 @@
 
 void arm_exception_handler(unsigned int *regs) {
 	panic("EXCEPTION:\n"
-		"r0=%08x r1=%08x r2=%08x r3=%08x\n"
-		"r4=%08x r5=%08x r6=%08x r7=%08x\n"
-		"r8=%08x r9=%08x r10=%08x r11=%08x\n"
-		"r12=%08x r14=%08x\n"
-		"cpsr=%08x spsr=%08x\n",
-		regs[2], regs[3], regs[4], regs[5],
-		regs[6], regs[7], regs[8], regs[9],
-		regs[10], regs[11], regs[12], regs[13],
-		regs[14], regs[15],
-	        regs[0], regs[1]);
+		  "r0=%08x r1=%08x r2=%08x r3=%08x\n"
```

Comment:
This change is a mystery for me. I see nothing wrong with 2 level indentation everywhere, but there are 2 extra spaces and 1 indentation level below